argparse.c

```diff
@@ -379,8 +379,16 @@ argparse_usage(struct argparse *self)
 
 int
 argparse_help_cb(struct argparse *self, const struct argparse_option *option)
+{
+    argparse_help_cb_no_exit(self, option);
+    exit(EXIT_SUCCESS);
+}
+
+int
+argparse_help_cb_no_exit(struct argparse *self,
+                         const struct argparse_option *option)
```

Comment:
It's a matter of style but I'd prefer to implement the calling function before the called one. Just because most people tend to read from top to bottom so IMHO it helps to already know what `argparse_help_cb_no_exit` does when seeing it being called in `argparse_help_cb`.